packages/desktop-client/src/components/reports/Reports.js

```diff
@@ -0,0 +1,454 @@
+          }}
+          isElement={isCashFlow}
+          isDefault={true}
+        />
+        <HeaderReport
+          title="Income & Expense"
+          id="iEHeader"
+          handleMouseHover={handleMouseHoverHeader}
+          handleMouseLeaveHeader={handleMouseLeaveHeader}
+          onHeaderClick={e => {
+            setIsTrends(true);
+            onHeaderClick(e, false);
+            onSelection(setIsIE);
+            setIsTotals(false);
+          }}
+          isElement={isIE}
+          isDefault={false}
+        />
+      </View>
+
+      <Header
+        allMonths={allMonths}
+        start={monthUtils.getMonth(start)}
+        end={monthUtils.getMonth(end)}
+        show1Month={!isNetWorth}
+        showAllTime={isIE}
+        onChangeDates={onChangeDates}
+        onApplyFilter={onApplyFilter}
+        onDeleteFilter={onDeleteFilter}
+        disableFilter={disableFilter}
+        filters={filterz}
+      />
+      <View
+        style={{
+          backgroundColor: 'white',
+          paddingLeft: 30,
+          paddingRight: 30,
+          overflow: 'auto',
+        }}
+      >
+        <View
+          style={{
+            flexShrink: 0,
+            flexDirection: 'row',
+            alignItems: 'flex-start',
+            paddingTop: 0,
+          }}
+        >
+          <View
+            style={{
+              backgroundColor: 'white',
+              overflow: 'auto',
+              flexGrow: 1,
+              padding: 10,
+            }}
+          >
+            <ChooseChartHeader
+              start={start}
+              end={end}
+              isNetWorth={isNetWorth}
+              isCashFlow={isCashFlow}
+              isIE={isIE}
+              isTotals={isTotals}
+              isTrends={isTrends}
+              totalIncome={totalIncome}
+              totalExpenses={totalExpenses}
+              totalChanges={totalChanges}
+              netWorth={netWorth}
+              handleClick={handleClick}
+              selectList={selectList}
+              handleChange={handleChange}
+            />
+
+            <ChooseChart
+              start={start}
+              end={end}
+              endDay={endDay}
+              graphData={graphData}
+              catData={catData}
+              isConcise={isConcise}
+              selectList={selectList}
+              isNetWorth={isNetWorth}
+              isCashFlow={isCashFlow}
+              isIE={isIE}
+              isTotals={isTotals}
+              isTrends={isTrends}
+            />
+          </View>
+          {isIE && (
+            <ChartExtraColumn
+              start={start}
+              end={end}
+              isIE={isIE}
+              totalIncome={totalIncome}
+              totalExpenses={totalExpenses}
+              selectList={selectList}
+            />
+          )}
+        </View>
+        <View style={{ marginTop: 30 }}>
+          {reportDescriptionTitle}
+          {reportDescription}
+        </View>
```

Comment:
I think these can be stored in one variable (or maybe better moved to a separate component entirely)

Reply:
Done.